### ISSUE TITLE
Does nothing on certain circumstances

### ISSUE THREAD

**Issue by toomasr** [cmrunton/tls-dashboard#3]
I set up the dashboard with couple of domains I know. The list was 

```
╰─$ cat node_app/monitored_hosts.js
var hosts = [
  'www.zeroturnaround.com',
  'www.zeroturnaround.org',
  'my.jrebel.com',
  'geekout.ee',
];
module.exports = hosts;
```

I ran the `node get_cert_info.js` and nothing happened. As I started debugging the script I found that the output is not written. I suspected that one of the domains didn't have a proper cert and I removed the zeroturnaround.org from the list and it started working.

Expected from the program:

Even if any of the domains has issues still write out results for others.

**Comment by cmrunton** [comment]
It looks like when I go to `www.zeroturnaround.org` I get redirected from HTTPS to HTTP, which would mean there is no peer certificate for the script to query. That's not a case I had anticipated, so I'll get it worked into the next fix. Thanks!

**Comment by toomasr** [comment]
Thanks!

**Comment by cmrunton** [comment]
Fixed with commit b095739. Let me know if you have any issues!
